src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.cs

```diff
@@ -6,7 +6,9 @@
 
 namespace Microsoft.Extensions.Logging
 {
+#if !NETFRAMEWORK
```

Comment:
While I understand why we don't want these to appear on .NET Framework, this is complicating more and more the build infrastructure around `SupportedOSPlatform` and `UnsupportedOSPlatform`. These `ref` files are usually autogenerated and the ifdefs are lost, so that adds too much burden to devs that want to autogenerate the ref to include a new API or ref change.